KS-1-5

RoboticRobotnik opened the pull request:
Transformed seconds into hh:mm:ss format

RoboticRobotnik committed:
KS-1-5
Transformed seconds into hh:mm:ss format

RoboticRobotnik commented:
Не нашел более адекватного способа добавить нули

ievetrov commented on src/main/kotlin/Lesson1/Task5.kt:
```diff
@@ -0,0 +1,12 @@
+
+fun main(){
+    val secundKosmos = 6480
+    val minutKosmos = secundKosmos/60
```
арифметические знаки следует обосабливать пробелами согласно конвенции стиля Котлин

ievetrov commented on src/main/kotlin/Lesson1/Task5.kt:
```diff
@@ -0,0 +1,12 @@
+    val chasKosmos = minutKosmos/60
+    val ostatokChasKosmos = minutKosmos%60
+
+    print("%02d".format(chasKosmos)+":"+"%02d".format(ostatokChasKosmos)+":"+"%02d".format(ostatokSecundKosmos))
```
правильный подход, но есть пару комментов
- мы используем Котлин, а он умеет в интерполяцию. поэтому лучше ее, чем складывать строки
- форматирование можно объединить вот примерно таким образом `println(String.format("%02d:%02d:%02d", hours, minutes, seconds))`

RoboticRobotnik committed:
KS-1-5
add changes according to convention
changed format
Transformed seconds into hh:mm:ss format
created empty package

ievetrov commented on src/main/kotlin/Lesson1/Task5.kt:
```diff
@@ -2,11 +2,11 @@ package Lesson1
+    val ostatokChasKosmos = minutKosmos % 60
 
-    print("%02d".format(chasKosmos)+":"+"%02d".format(ostatokChasKosmos)+":"+"%02d".format(ostatokSecundKosmos))
+    print(String.format("%02d:%02d:%02d",chasKosmos,ostatokChasKosmos,ostatokSecundKosmos))
```
хорошо! давай только еще добавим пробелы после запятых для соблюдения стиля

RoboticRobotnik committed:
KS-1-5
add changes according to convention
changed format
Transformed seconds into hh:mm:ss format
created empty package

ievetrov reviewed (approved):
все красиво!
